lib/npm/test.js

```diff
@@ -55,8 +55,16 @@ function test(context, next) {
       npmBin = path.join(path.dirname(npmBin), 'node_modules', 'npm', 'bin',
           'npm-cli.js');
     }
-    const proc = spawn(nodeBin, [npmBin, 'test', '--loglevel=' +
-        context.options.npmLevel], options);
+
+    let args = [npmBin, 'test', '--loglevel=' + context.options.npmLevel];
+    if (context.options.coverage) {
+      args = [__dirname + './../../node_modules/nyc/bin/nyc.js',
```

Comment:
Generally +1 but don't you need to add nyc into the package.json as a dependency for CiTGM ?